src/app/services/user.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from "@angular/core";
+import { AngularFirestore, AngularFirestoreCollection, AngularFirestoreDocument } from "@angular/fire/compat/firestore";
+import { Observable } from "rxjs";
+
+
+
+@Injectable({
+    providedIn: 'root',
+})
+
+export class UserService {
+    private UserCollection: AngularFirestoreCollection<any>;
+    private Users: Observable<any[]>;
```

Comment:
Observables use Hungarian notation (suffixed by $ to indicate it's an observable).
Use of any.